index.html

```diff
@@ -0,0 +1,310 @@
+                            
+                            <ul class="card-tags">
+                                <li class="card-tag ruby"><p class="card-tag-text">Ruby on rails</p></li>
+                                <li class="card-tag css"><p class="card-tag-text">css</p></li>
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
+                            </div>
+                        </section>
+                    </div>
+
+                    <!-- card3 -->
+                    <div class="card">
+                        <section class="card-content">
+                            <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+                            
+                            <ul class="card-tags">
+                                <li class="card-tag ruby"><p class="card-tag-text">Ruby on rails</p></li>
+                                <li class="card-tag css"><p class="card-tag-text">css</p></li>
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
+                            </div>
+                        </section>
+                    </div>
+                </section>
+            </div>
+        </section>
+        
+        <!-- About me Section -->
+        <section id="about">
+            <div id="about-heading">    
+                <h2 id="about-title">About me</h2>
+
+                <p class="about-description"> I can help you build a product , 
+                    feature or website Look through 
+                    some of my work and experience! 
+                    If you like what you see and have 
+                    a project you need coded, 
+                    don't hestiate to contact me.</p>
+                <a href="resume.pdf" download="My Resume"><button class="about-resume-button green-button"><p id="about-resume-button-text">Get My Resume</p></button></a>
+                <img src="img/scattered-backgrounds/about-me-mobile-background1.svg" alt="" class="about-me-mobile1">
+                <img src="img/scattered-backgrounds/about-me-mobile-background2.svg" alt="" class="about-me-mobile2">
+                <img src="img/scattered-backgrounds/bottom-most-mobile-background.svg" alt="" class="bottom-most-mobile-background">
+            </div>
+            
```

Comment:
- [ ] I think you are missing the background image for the about section, please get the background images from figma and add it to the appropriate sections
what you have :point_down: 
![image](https://user-images.githubusercontent.com/97521557/220353548-b77852ae-74df-40a3-9510-296f7f4fdc8e.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/220353589-60bb4c93-3d98-4d16-a8a9-0c030c8a1dea.png)